tests/calculations/test_md.py

```diff
@@ -0,0 +1,209 @@
+"""Tests for geometry optimisation calculation."""
+
+import os
+import subprocess
+
+from ase.build import bulk
+from ase.io import read
+import pytest
+
+from aiida.common import datastructures
+from aiida.engine import run_get_node
+from aiida.orm import Dict, Str, StructureData
+from aiida.plugins import CalculationFactory
+
+from aiida_mlip.data.config import JanusConfigfile
+from aiida_mlip.data.model import ModelData
+
+
+def test_MD(fixture_sandbox, generate_calc_job, janus_code, model_folder):
+    """Test generating MD calculation job."""
+
+    entry_point_name = "janus.md"
+    model_file = model_folder / "mace_mp_small.model"
+    inputs = {
+        "metadata": {"options": {"resources": {"num_machines": 1}}},
+        "code": janus_code,
+        "arch": Str("mace"),
+        "precision": Str("float64"),
+        "struct": StructureData(ase=bulk("NaCl", "rocksalt", 5.63)),
+        "model": ModelData.local_file(model_file, architecture="mace"),
+        "device": Str("cpu"),
+        "ensemble": Str("nve"),
+        "md_kwargs": Dict(
+            {
+                "temp": 300.0,
+                "steps": 4,
+                "traj-every": 1,
+                "restart-every": 3,
+                "stats-every": 1,
+            }
+        ),
+    }
+
+    calc_info = generate_calc_job(fixture_sandbox, entry_point_name, inputs)
+
+    cmdline_params = [
+        "md",
+        "--arch",
+        "mace",
+        "--struct",
+        "aiida.xyz",
+        "--device",
+        "cpu",
+        "--log",
+        "aiida.log",
+        "--summary",
+        "md_summary.yml",
+        "--calc-kwargs",
+        f"{{'default_dtype': 'float64', 'model': '{model_file}'}}",
+        "--ensemble",
+        "nve",
+        "--temp",
+        300.0,
+        "--steps",
+        4,
+        "--traj-every",
+        1,
+        "--stats-every",
+        1,
+        "--restart-every",
+        3,
+        "--traj-file",
+        "aiida-traj.xyz",
+        "--stats-file",
+        "aiida-stats.dat",
+    ]
```

Comment:
Would it be better to list these as pairs? Or does black complain?

Reply:
it complains